config/validated_config_db_connector.py

```diff
@@ -0,0 +1,37 @@
+import jsonpatch
+from jsonpointer import JsonPointer
+
+from generic_config_updater.generic_updater import GenericUpdater, ConfigFormat
+
+def ValidatedConfigDBConnector(config_db_connector):
+    config_db_connector.set_entry = validated_set_entry
+    return config_db_connector
+
+def make_path_value_jsonpatch_compatible(table, key, value):
+    if type(key) == tuple:
+        path = JsonPointer.from_parts([table, '|'.join(key)]).path
+    else:
```

Comment:
>else [](http://example.com/codeflow?start=4&length=4)

In this branch, you can also leverage `JsonPointer.from_parts`, and actually compare with above block, you can do fine granularity if/else.

Reply:
Changed to use `JsonPointer.from_parts` in both branches